### PR TITLE
Hanna - adding in spoontacular API calls

### PR DESCRIPTION
**Pull request by hannamunoz**
Merging recipe API changes into main branch. We still need to display recipes coming back from API response. You can view the API response/structure in the browser network tab. The response object has an image, title, link to recipe, and a few other things.

### CODE REVIEW

File: assets/js/recipe.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    // all form data is in dataObj now
+    console.log(dataObj);
+    searchRecipes(dataObj.diet, dataObj.ingredients.toString(), dataObj.intolerances.toString());
+});
+
+// need a function to string text from input groups to add to api request
+function grabInputs() {
+
+}
+
```
Comment (TheTiiiim) [inline assets/js/recipe.js:49]: function is empty and not callerd anywhere

File: assets/js/recipe.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+function searchRecipes(diet, includeIngredients, intolerances) {
+    const numberOfRecipes = 5;
+    const dietRestriction = (diet.toLowerCase() === 'regular diet') ? '' : diet;
+    let queryURL = `https://api.spoonacular.com/recipes/complexSearch?diet=${dietRestriction}&intolerances=${intolerances}&includeIngredients=${includeIngredients}&number=${numberOfRecipes}&addRecipeInformation=true&apiKey=${apiKey}`;
+
+    $.ajax({
+        url: queryURL,
+        method: "GET",
+        success: (res) => {
+            return res;
```
Comment (TheTiiiim) [inline assets/js/recipe.js:61]: returning a value within a promise callback returns `undefined`, return the promise and use a `.then()` after the function call to deal with data

**Review by TheTiiiim — CHANGES_REQUESTED** [review]
also delete mealSearchForm submission from `script.js`

**Comment by hannamunoz** [comment]
> also delete mealSearchForm submission from `script.js`

this should be deleted, can you verify?

**Review by TheTiiiim — APPROVED** [review]

**Review by Taylorgonz — APPROVED** [review]

**Pull request merged**